scripts/release/update-version.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const glob = require('glob');
+
+// read version from input arg
+const version = process.argv[2];
+// abort if no version is given
+if (!version) {
+  console.error('No version specified');
+  process.exit(1);
+}
+
+// update package.json version
+const packageJson = JSON.parse(fs.readFileSync('package.json', 'utf8'));
+packageJson.version = version;
+fs.writeFileSync('package.json', JSON.stringify(packageJson, null, 2));
+
+const spdxString = '// SPDX-License-Identifier: MIT';
+const versionPrefix = '// ERC721A Contracts v';
+
+// loop through all files with contracts/**/*.sol pattern
+glob('contracts/**/*.sol', null, function (err, files) {
+  files.forEach((file) => {
+    // read file content
+    const content = fs.readFileSync(file, 'utf8');
+
+    const versionStringLine = versionPrefix + version;
+
+    let updatedContent;
+    if (content.includes(versionPrefix)) {
+      updatedContent = content.replace(new RegExp(`${versionPrefix}.*`), `${versionStringLine}`);
+    } else {
+      updatedContent = content.replace(new RegExp(`${spdxString}`), `${spdxString}\n${versionStringLine}`);
```

Comment:
Why is this in the else? Would'nt we always want to change the `spdxString` incase this script is used to update the License as well?

Reply:
This script doesn't ment to update the License. The usage of `spdxString` is for the contracts that don't have a version line.  

Suppose a contract already has a version line. In that case, we are just updating that line, but some contracts may be added later and don't have a version string; for those, we are using `spdxString` and adding the version line after it.